src/user/entity/user.entity.ts

```
@@ -1,7 +1,7 @@
-import { Calender } from 'src/calender/entity/calender.entity';
+import { Calendar } from 'src/calendar/entity/calendar.entity';
 import { ROLE } from 'src/common/enum/role';
 import { Promise } from 'src/promise/entity/promise.entity';
```

Biome (1.9.4)

[error] Do not shadow the global "Promise" property.

Consider renaming this variable. It's easy to confuse the origin of variables when they're named after a known global.

(lint/suspicious/noShadowRestrictedNames)